sdk/appconfiguration/AzureAppConfiguration/Source/Models/ConfigurationSetting.swift

```diff
@@ -54,7 +54,7 @@ public struct ConfigurationSetting: Codable {
         self.label = label
         self.tags = tags
         self.contentType = contentType
-        self.etag = eTag
+        etag = eTag
```

Comment:
Here's an example where swiftlint should not have made this change. Very odd. I will check out your branch and try running it locally.

Reply:
Just tried this and swiftformat undid a bunch of changes including this one.

Reply:
Hmm...yeah, I see what you mean. I just pushed that simple change you recommended about the alias and locally ran `swiftformat`. It made a bunch of changes. So it seems that its inclusion in the `swiftlint.sh` script isn't always working as intended.